core/rio/languages/src/main/java/org/eclipse/rdf4j/rio/languages/BCP47LanguageHandler.java

```diff
@@ -0,0 +1,101 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Eclipse RDF4J contributors, Aduna, and others.
```

Comment:
I was not completely sure about changing it, given it started as a straight copy-paste of the RFC3066 file, with minor changes, and hence to me it seems like it should inherit the header from that file. However, I have changed it, can revert if necessary.